src/CliMenu.php

```diff
@@ -200,6 +200,22 @@ private function selectFirstItem() : void
         }
     }
 
+    /**
+     * Disables the built-in VIM control mappings
```

Comment:
Yeah I don't think we should disable the default arrow keys, just the extra vim things?

Reply:
what do you mean? this doesn't disable arrow keys.. least not on my keyboard

Reply:
I'm on OSX, this doesn't affect arrow keys. Is this a Linux thing (^K, ^P == arrows?) Do you prefer if I unset the 4 mappings mapped to k,j,l,m ?

Reply:
@AydinHassan @mikeymike 
Controls like arrows are defined in InputCharacter from Terminal if I'm not mistaken so that should only removes "special" controls

Reply:
ah okay cool! sorry misunderstood.